assets/js/blocks/price-filter/types.ts

```diff
@@ -6,3 +6,7 @@ export interface Attributes {
 	headingLevel: number;
 	className?: string;
 }
+export interface PriceFilterBlockProps {
+	attributes: Attributes;
+	isEditor: boolean;
+}
```

Comment:
Instead of creating a separate interface, what are your thoughts on adding `isEditor: boolean;` to `Attributes` and renaming that interface to `PriceFilterBlockProps`? The final interface could look like this:
```ts
export interface Attributes {
	showFilterButton: boolean;
	showInputFields: boolean;
	inlineInput: boolean;
	heading: string;
	headingLevel: number;
	className?: string;
	isEditor?: boolean;
}
```

In this updated interface, I've added `isEditor?: boolean;` as an optional value to prevent introducing new TS errors in files that import the initial interface.

Of course, when we'd rename the interface from `Attributes` to `PriceFilterBlockProps`, we'd also have to adjust the use in the other files. 

What are your thoughts on these suggestions, @hritikchaudhary?

Reply:
> In this updated interface, I've added `isEditor?: boolean;` as an optional value to prevent introducing new TS errors in files that import the initial interface.

Yes this will also work.
> 
> Of course, when we'd rename the interface from `Attributes` to `PriceFilterBlockProps`, we'd also have to adjust the use in the other files.
> 
> What are your thoughts on these suggestions, @hritikchaudhary?

@nielslange  Due to these reasons only I've kept two interfaces.. as `Attributes` might be used in other files so I didn't touch that.. so It will not cause any TS errors in other files and `PriceFilterBlockProps` is not imported by any file other than block.tsx.
Another reason is that `Attributes` properties will remain intact, and if in future we want to add some new properties which doesn't align with `Attribute` can go in `PriceFilterBlockProps`, but that is just for readability it'll not have any positive or negative impact on functionality, I'm good either way.

Reply:
@hritikchaudhary I had a quick chat with @opr on this, and he also thinks that it's better to add `isEditor?: boolean;` to the current interface called `Attributes` and then rename it to `PriceFilterBlockProps` and adjust the imports accordingly in the other files. While these are a few extra steps, it makes the codebase more consistent. It would be great if you could adjust that.

Reply:
Yes sure I'll push this change